tests/scanner/scanner_test.py

```diff
@@ -35,92 +34,100 @@
-        + '/google/cloud/security/scanner'
-        self.scanner.FLAGS.use_engine = 'iam_rules_engine.py'
-        self.scanner.LOGGER.warn = mock.MagicMock()
-        with self.assertRaises(SystemExit):
-            self.scanner.main(self.fake_main_argv)
-
-    # TODO: Fix this test
-    #@mock.patch.object(ire.IamRulesEngine, 'build_rule_book', autospec=True)
-    #@mock.patch.object(scanner, '_get_timestamp')
-    #def test_no_timestamp_raises_systemexit(self, mock_get_timestamp, mock_build_rule_book):
-    #    """Test that no org or project policies raises SystemExit/calls sys.exit()."""
-    #    mock_get_timestamp.return_value = None
-    #    with self.assertRaises(SystemExit):
-    #        self.scanner.main(self.fake_main_argv)
-    #    self.assertEqual(1, mock_build_rule_book.call_count)
-    #    self.scanner.LOGGER.warn.assert_called_with('No snapshot timestamp found. Exiting.')
-
-    # TODO: Fix this test
-    #@mock.patch.object(ire.IamRulesEngine, 'build_rule_book')
-    #@mock.patch.object(scanner, '_get_timestamp')
-    #@mock.patch.object(irep.LoadIamDataPipeline, '_get_org_policies')
-    #@mock.patch.object(irep.LoadIamDataPipeline, '_get_project_policies')
-    #def test_no_policies_raises_systemexit(
-    #        self,
-    #        mock_project_policies,
-    #        mock_org_policies,
-    #        mock_get_timestamp,
-    #        mock_build_rule_book):
-    #    """Test that no org or project policies raises SystemExit/calls sys.exit()."""
-    #    mock_get_timestamp.return_value = self.fake_timestamp
-    #    mock_org_policies.return_value = []
-    #    mock_project_policies.return_value = []
-    #    with self.assertRaises(SystemExit):
-    #        self.scanner.main(self.fake_main_argv)
-    #    self.scanner.LOGGER.warn.assert_called_with('No policies found. Exiting.')
-
-    # TODO: Fix this test
-    #@mock.patch.object(ire.IamRulesEngine, 'build_rule_book', autospec=True)
-    #@mock.patch.object(scanner, '_get_timestamp')
-    #@mock.patch.object(scanner, '_get_org_policies')
-    #@mock.patch.object(scanner, '_get_project_policies')
-    #@mock.patch.object(irep.LoadIamDataPipeline, 'find_violations')
-    #@mock.patch.object(scanner, '_output_results')
-    #def test_main_no_violations(self,
-    #        mock_output_results,
-    #        mock_find_violations,
-    #        mock_project_policies,
-    #        mock_org_policies,
-    #        mock_get_timestamp,
-    #        mock_build_rule_book):
-    #    """Test main()."""
-    #    mock_get_timestamp.return_value = self.fake_timestamp
-    #    mock_org_policies.return_value = {'a': 1}
-    #    mock_project_policies.return_value = {'b': 2}
-    #    mock_find_violations.return_value = ['a']
-    #    self.scanner.main(self.fake_main_argv)
-    #    self.assertEqual(1, mock_output_results.call_count)
-
-    # TODO: fix this test
-    #def test_find_violations(self):
-        """Test that find_violations() is called.
+  FAKE_FORSETI_CONFIGS = {
```

Comment:
Looks like there's a lot of changes in this file, but it's whitespace (indentation) -- could you make sure the indent is 4 spaces per indent?